content/en-us/studio/texture-generator.md

```diff
@@ -193,18 +193,21 @@ The following prompting tips enable the tool to provide better and more consiste
 - Use descriptive phrases instead of single words for key items, for example "worn&nbsp;leather" instead of just "leather."
 - Add styling cues like "cinematic," "photorealistic," "cartoon," or "low&nbsp;poly" to influence the overall look.
 </TabItem>
+
 <TabItem label="Generation Angle">
 The [generation angle](#generation-angle) sets the most important view of your mesh which will be prioritized during generation. Changing this angle can often help eliminate inconsistencies in textures.
 
 - Setting the generation angle to highlight the meaningful faces of your mesh will result in better generations.
 - For flat objects like plates, swords, and fences, make sure the generation angle displays a view with a high surface area.
 - For humanoid and animal meshes, experiment with different generation angles, such as a head-on view of the face and a side view profile, to find the most consistent and coherent texture.
 </TabItem>
+
 <TabItem label="Iterative Refinement">
 Generating the most suitable textures is an iterative process. It helps to preview often and tweak prompts incrementally to reach your vision.
 
 - If a texture doesn't meet your expectations, identify what to change instead of starting over. For example, tweak the color or pattern description.
 - Change the prompt word order. Words at the beginning of the prompt can have more weight when generating.
 - Generate several previews to compare different prompts, or try the same prompts with different [seeds](#seed-control) to get a preview that fits your vision.
 </TabItem>
+
```

Comment:
Intentional? Does it change anything?

Reply:
I believe this was a merge of the branch main of this repository and the branch main of my fork.

Reply:
If this branch is up to date, it shouldn't show any diff to main

Reply:
Huh, I don't remember editing that.

Reply:
Looks like it's from here https://github.com/Roblox/creator-docs/commit/034e32d44d463b27ed7c1f33f5dd61a3209c5c52

Reply:
I believe it may have been automatic, when I made the pull request, there were some linting issues and I ran a command in the console which fixed them (can't remember which command right now), I'll take a look though.